lib/app_manifest/environment.rb

```diff
@@ -0,0 +1,21 @@
+module AppManifest
+  class Environment
+    include Virtus.model
+    include Serializer
+
+    attribute :addons,      Array[Addon]
```

Comment:
Given that this repeats the attributes from the base schema, it might be worth pulling them out into a common helper or something so that we can be sure the attribute lists stay in sync. Thoughts?